StRoot/StMuDSTMaker/COMMON/StMuDstMaker.cxx

```diff
@@ -479,6 +479,9 @@ void  StMuDstMaker::streamerOff() {
   StMuL3EventSummary::Class()->IgnoreTObjectStreamer();
 #ifndef __NO_STRANGE_MUDST__
   StStrangeMuDst::Class()->IgnoreTObjectStreamer();
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6,0,0) 
+  StStrangeAssoc::Class()->IgnoreTObjectStreamer(); 
+#endif
```

Comment:
I think it is the right place for this but don't we really need to hid this from ROOT5? I think it should work fine for ROOT5